awx_collection/plugins/modules/project_update.py

```diff
@@ -114,7 +114,9 @@ def main():
     # Update the project
     result = module.post_endpoint(project['related']['update'])
 
-    if result['status_code'] != 202:
+    if result['status_code'] == 405:
+        module.fail_json(msg="Unable to trigger a project update because the project scm_type ("+project['scm_type']+") does not support it.", response=result)
```

Comment:
Let's use `format()` to be more consistent with the rest of the codebase.

```suggestion
        module.fail_json(
            msg="Unable to trigger a project update because the project scm_type ({0}) does not support it.".format(project['scm_type']),
            response=result
        )
```